content/pages/contributing.md

```diff
@@ -0,0 +1,427 @@
+# Contributor's Guide
+
+## Overview
+
+Welcome! This is the main contributors guide for Project Pythia.
+Project Pythia is an open community, and all contributions are
+welcome following our [Code of
+Conduct](https://projectpythia.org/pages/code_of_conduct.html). All
+of the content for Project Pythia is hosted on GitHub in a number
+of different public repositories. From this document you can learn
+about the many ways that you can contribute to this community
+project, and where you can find additional information. In many
+cases pertinent details may be found in repository-specific
+contributors guides **that should always be consulted in addition
+to this guide**. More on this topic later.
+
+## The many ways to contribute
+
+### Starting a conversation
+
+One of the easiest ways to contribute is to communicate with the
+Project Pythia community. You might, for example, suggest changes
+to content on the site, ask a clarifying question, or even respond
+to another community member’s question. There are a number of ways
+to have a dialogue with the Pythia community, such as by opening a
+GitHub issue, but the easiest method is to simply create (or respond
+to) a post on our [discussion
+forum](https://github.com/ProjectPythia/projectpythia.github.io/discussions).
+
+### Add content to the External Links Gallery
+
+The Project [Pythia External Links
+Gallery](https://projectpythia.org/pages/links.html) contains a
+collection of links to useful resources that were created and are
+hosted outside of Project Pythia. Some examples include: documentation
+and tutorials for iconic packages such as Matplotlib, Cartopy, and
+Xarray; on-line courses from universities and other educational
+institutions; and Jupyter Notebook galleries assembled and maintained
+by a variety of other sites. Project Pythia makes it possible to
+filter and search the External Links Gallery in a number of ways
+to ease finding relevant content. We encourage the community to
+consider submitting your own geoscience education material links
+to Project Pythia.
+
+Contributing a new link is easy with these three simple steps:
+
+1. Navigate your web browser to the main Project Pythia repo's “issue” page [here](https://github.com/ProjectPythia/projectpythia.github.io/issues).
```

Comment:
These steps are different/there are 2 paths now that #87 is merged. The user can also:

1. Navigate to the External Links Gallery.
2. Click the "Submit a Link" button, which will redirect them to a new PR template.
3. Fill out that template with the information relevant to their Python learning resource.

Reply:
Nice! I updated the documentation per above.

Reply:
Great!